app/src/organisms/Desktop/Devices/ProtocolRun/ProtocolRunCamera/ImageGalleryContainer/GalleryItemCard.tsx

```diff
@@ -70,17 +164,20 @@ export function GalleryItemCard({
           </StyledText>
         </div>
       </div>
+
       <div className={styles.gallery_card_cmd_txt_container}>
-        <StyledText desktopStyle="bodyDefaultRegular">
-          {stepCommandText}
+        <StyledText desktopStyle="bodyDefaultRegular" color={COLORS.black90}>
```

Comment:
```suggestion
        <StyledText desktopStyle="bodyDefaultRegular">
```
I believe black90 is the default color and pretty much what we use everywhere, so we can safely omit it 👍 

Even though bodyDefaultRegular is the default desktop style, it's still nice to keep it for the explicitness, IMO.